src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -7,14 +7,13 @@
 
 import d3a.constants
 from d3a.d3a_core.exceptions import (
-    InvalidBidOfferPairException, OfferNotFoundException,
-    BidNotFoundException, MycoValidationException)
+    InvalidBidOfferPairException, MycoValidationException)
 from d3a.d3a_core.redis_connections.redis_area_market_communicator import ResettableCommunicator
 from d3a.models.market import Market
 
 
 class ExternalMatcherEventsEnum(Enum):
```

Comment:
Nit: Not so relevant, but I think we can also avoid adding the `Enum` suffix to the class name.
```suggestion
class ExternalMatcherEvents(Enum):
```